vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -700,13 +697,11 @@ void AdsImpl::OnPageLoaded(
     return;
   }
 
-  if (TestSearchState(url)) {
+  if (SearchProviders::IsSearchEngine(url)) {
     BLOG(INFO) << "Site visited " << url << ", URL is a search engine";
```

Comment:
should the URL even be logged? logs can get persisted to disk unintentionally

Reply:
yea, this seems like it should be a VLOG entry anyway if we kept it at all. `INFO` doesn't seem like the right logging level even without the persistence issue